test-project/Assets/.Schema/from_gdk_packages/com.improbable.gdk.core/common.schema

```diff
@@ -0,0 +1,3 @@
+package improbable.common;
```

Comment:
Why is this necessary?

Reply:
oh that was added accidentally, will remove it and check the gitignore

Reply:
I've updated the gitignore in another PR - rebase and you should be good